apps/crossroads_interface/web/templates/layout/no_header_or_footer.html.eex

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="crossroads">
```

Comment:
Will this be an issue for other MC's using this layout?  I saw that it was migrated from "legacy," so just wanted to check.